framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/widgets/RecentBlocks.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import * as React from 'react';
+import Widget from '../Widget';
+
+interface Props {
+	onClick?: (event: Event | React.MouseEvent<HTMLTableRowElement>) => void;
+}
+
+const RecentBlocks: React.FC<Props> = props => (
```

Comment:
if this is created, maybe it's better to add it i nthe main page to see

Reply:
It was just sample to show how to use `Widget`, it still need styling for the table to make it look like exactly the same as design. I think one who will actually implement it should add it to any page that time.